<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroducirPlanta</class>
 <widget class="QDialog" name="IntroducirPlanta">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IntroducirPlanta</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>549</width>
     <height>284</height>
    </rect>
   </property>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>521</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="fT">
         <property name="text">
          <string>Fu&amp;nci√≥n de Transferencia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Formato_Libre">
         <property name="text">
          <string>Formato Libre</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="pYCe">
         <property name="text">
          <string>Polos &amp;y Ceros</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cDPol">
         <property name="text">
          <string>C. de Polinomios</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="kGa">
         <property name="text">
          <string>&amp;k = hfgain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="kNoGa">
         <property name="text">
          <string>k= &amp;lfgain</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="agrupador">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QWidget" name="vacia"/>
    <widget class="QWidget" name="formato_libre">
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>217</width>
        <height>128</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Numerador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numeFL"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Denominador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="denoFL"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Ganancia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kFL"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Retardo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="retFL"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="CP">
     <widget class="QLabel" name="imaCP">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Numerador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numeCP"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Denominador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="denoCP"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ganancia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kCP"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Retardo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="retCP"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="kGa_2">
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Numerador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numeKG"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Denominador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="denoKG"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Ganancia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kKG"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Retardo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="retKG"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="imaKG">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="kNG">
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Numerador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numeKNG"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Denominador:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="denoKNG"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Ganancia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="kkNG"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Retardo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="retKNG"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="imaKNG">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>171</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nombre"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Inertidumbre">
       <property name="text">
        <string>Incertidumbre</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>11</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
